.devcontainer/db/Dockerfile

```
@@ -27,6 +26,11 @@ COPY ./.devcontainer/development/config/shared/my.cnf /etc/mysql/my.cnf
 RUN chmod 744 /etc/mysql/my.cnf
 RUN chmod 755 /scripts
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)